### ISSUE TITLE
[Windows] thyme error when run `thyme track`

### ISSUE THREAD

**Issue by morxs** [sourcegraph/thyme#64]
Hi Dev,
I downloaded the 0.2.2 windows binary, and when run thyme show error on windows 10 64bit

```
Exception 0xc0000005 0x1 0x450654 0x7594db55
PC=0x7594db55

syscall.Syscall(0x7594da60, 0x1, 0x203f8, 0x0, 0x0, 0x13614000, 0x0, 0x136401a0)
        /home/beyang/go/src/runtime/syscall_windows.go:163 +0x4a
syscall.(*Proc).Call(0x1363a6c0, 0x1363a6b4, 0x1, 0x1, 0x4, 0x573ba0, 0x0, 0x0)
        /home/beyang/go/src/syscall/dll_windows.go:147 +0x119
syscall.(*LazyProc).Call(0x13688a20, 0x1363a6b4, 0x1, 0x1, 0x1, 0x1363a698, 0x0, 0x0)
        /home/beyang/go/src/syscall/dll_windows.go:304 +0x54
github.com/sourcegraph/thyme.getWindowID(0x203f8, 0x1363a6b0, 0x0)
        /home/beyang/src/github.com/sourcegraph/thyme/windows.go:59 +0x5b
github.com/sourcegraph/thyme.(*WindowsTracker).Snap.func1(0x203f8, 0x378, 0xdfeac)
        /home/beyang/src/github.com/sourcegraph/thyme/windows.go:95 +0x1f8
syscall.Syscall(0x75963a70, 0x2, 0x451d60, 0x378, 0x0, 0x13614000, 0x0, 0x13604380)
        /home/beyang/go/src/runtime/syscall_windows.go:163 +0x4a
syscall.(*Proc).Call(0x1363a650, 0x1363a628, 0x2, 0x2, 0x8, 0x5745e0, 0x0, 0x0)
        /home/beyang/go/src/syscall/dll_windows.go:149 +0x28c
syscall.(*LazyProc).Call(0x136889c0, 0x1363a628, 0x2, 0x2, 0x5b0da4, 0x0, 0x0, 0x0)
        /home/beyang/go/src/syscall/dll_windows.go:304 +0x54
github.com/sourcegraph/thyme.(*WindowsTracker).Snap(0x668ec4, 0x668ec4, 0x0, 0x0)
        /home/beyang/src/github.com/sourcegraph/thyme/windows.go:110 +0x211
main.(*TrackCmd).Execute(0x656f00, 0x1363e418, 0x0, 0x1, 0x0, 0x0)
        /home/beyang/src/github.com/sourcegraph/thyme/cmd/thyme/main.go:40 +0x76
github.com/jessevdk/go-flags.(*Parser).ParseArgs(0x1363c540, 0x1363a0e8, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/beyang/src/github.com/jessevdk/go-flags/parser.go:314 +0x917
github.com/jessevdk/go-flags.(*Parser).Parse(0x1363c540, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/beyang/src/github.com/jessevdk/go-flags/parser.go:185 +0x78
main.main.func1(0x0, 0x0)
        /home/beyang/src/github.com/sourcegraph/thyme/cmd/thyme/main.go:141 +0x2d
main.main()
        /home/beyang/src/github.com/sourcegraph/thyme/cmd/thyme/main.go:148 +0x22
eax     0xe50
ebx     0x203f8
ecx     0x513721bc
edx     0x0
edi     0x43ddbb20
esi     0x450654
ebp     0xdfdf0
esp     0xdfddc
eip     0x7594db55
eflags  0x10246
cs      0x23
fs      0x53
gs      0x2b
```

**Comment by Olreich** [comment]
This should be fixed by #59. Whenever a new release goes out this issue should go away. Otherwise, you can `go get -u github.com/sourcegraph/thyme` if you have Go installed, then use the binary you get out at `%GOPATH%/bin/thyme.exe`

**Comment by morxs** [comment]
Thanks... updated using `go get` and now no issue anymore
